vars/mavenExecute.groovy

```diff
@@ -114,11 +120,15 @@ void call(Map parameters = [:]) {
             command += " ${mavenGoals}"
         }
         def defines = configuration.defines
-        if (defines?.trim()){
+        if (defines?.trim()) {
             command += " ${defines}"
         }
+
         dockerExecute(script: script, dockerImage: configuration.dockerImage, dockerOptions: configuration.dockerOptions) {
-            sh command
+            commandOutput = sh(returnStdout: true, script: command)
```

Comment:
Having `returnStdout` set to `true` means the output delivered via `stdout` does not appear in the log. It is delivered to `commandOutput`. 

We are now responsible for writing it into the log. I guess that's bad. Other (... normal maven calls for performing a project build)  will not have the maven log contained in the job log (... convince me from the oposite), despite we put an corresponding `echo` statement into our code. That is a dis-improvement. Even if we echo `commandOutput` into the log we have the disadvantage that there will be no output during the maven build run, everything will be delivered in one step after the build (when we perform an echo statement accordingly). I personally hate it when build tools does not tell me what they are doing ...

I would like to suggest to use the `configuration.returnStdout` flag also here rather than hard coding it.